Dockerfile

```
@@ -1,22 +1,31 @@
 # To run: docker run -v /path/to/wsgi.py:/var/www/sheepdog/wsgi.py --name=sheepdog -p 81:80 sheepdog
-# To check running container: docker exec -it sheepdog /bin/bash 
+# To check running container: docker exec -it sheepdog /bin/bash
 
-FROM quay.io/cdis/py27base:pybase2-1.0.2
+FROM quay.io/cdis/python-nginx:pybase3-1.0.0
```

```suggestion
FROM quay.io/cdis/python-nginx:pybase3-1.1.0
```
let's use the possibly-fastest version